README.md

```diff
@@ -5,11 +5,30 @@ The DNS Operator deploys and manages [CoreDNS](https://coredns.io) to provide a
 The operator tries to be useful out of the box by creating a working default deployment based on the cluster's configuration.
 
 * The default cluster domain is `cluster.local`.
-* Configuration of the CoreDNS [Corefile](https://coredns.io/manual/toc/#configuration) or [kubernetes plugin](https://coredns.io/plugins/kubernetes/) is not yet supported.
+* Limited configuration of the CoreDNS [Corefile](https://coredns.io/manual/toc/#configuration) or [kubernetes plugin](https://coredns.io/plugins/kubernetes/) is supported.
```

Comment:
@Miciah I'm confused by `or` in this sentence. The k8s plugin is a subset of the corefile config, right?

Reply:
Right, I should change "or" to "and".  Would that be sufficient?

Reply:
I would keep the bullet as:

Limited configuration of the CoreDNS [Corefile](https://coredns.io/manual/toc/#configuration).